lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -443,6 +443,9 @@
               <p><?php print $voter_reg_share_modal_text_1 ?></p>
```

Comment:
I think we could take this as a chance to clean up this code and remove the voter modal